perses/tests/test_atom_mapping.py

```diff
@@ -21,8 +21,8 @@ class TestAtomMapping(unittest.TestCase):
     """Test AtomMapping object."""
     def setUp(self):
         """Create useful common objects for testing."""
-        self.old_mol = Molecule.from_smiles('[C:0]([H:1])([H:2])([H:3])[C:4]([H:5])([H:6])([H:7])') # ethane
-        self.new_mol = Molecule.from_smiles('[C:0]([H:1])([H:2])([H:3])[C:4]([H:5])([H:6])[O:7][H:8]') # ethanol
+        self.old_mol = Molecule.from_mapped_smiles('[C:1]([H:2])([H:3])([H:4])[C:5]([H:6])([H:7])([H:8])') # ethane
```

Comment:
So does the mapping index start at `1` instead of `0` now?

Reply:
Yup - It's 1-indexed! https://docs.openforcefield.org/projects/toolkit/en/stable/api/generated/openff.toolkit.topology.Molecule.html?highlight=from_mapped_smiles#openff.toolkit.topology.Molecule.from_mapped_smiles

I will note that `from_mapped_smiles` guarantees that the output molecule will have the atoms INDEXED in the map order, but the output molecule won't have anything in `mol.properties['atom_map']`. On the other hand, running boring old `from_smiles` with a mapped smiles doesn't guarantee anything about the output atom indexing, but does populate the appropriate mapping in `molecule.properties['atom_map']`.

For testing purposes (where you need atom 3 to be a hydrogen, for example) it's usually correct to use `from_mapped_smiles`. But since I know perses uses atom maps for other purposes, I wanted to double check that this test doesn't need anything in the `molecule.properties['atom_map']` dict :-)

Reply:
I did a search for `atom_map` being used as a key for anything in perses and I didn't find any matches. I think we should be safe that way. Thank you!